packages/bootstrap/scss/multiselect/_theme.scss

```diff
@@ -1,28 +1,25 @@
 @include exports("multiselect/theme") {
 
     // Appearance
-    .k-multiselect {
+    .k-multiselect-wrap {
         @include appearance( input );
         transition: $transition;
 
+        &:hover,
         &.k-state-hover {
             @include appearance( hovered-input );
         }
 
+        &:focus,
         &.k-state-focused {
             .k-multiselect-wrap {
```

Comment:
This rule searches for an element that is `.k-multiselect-wrap.k-state-focused > .k-multiselect-wrap`, which doesn't exist, and breaks the focused state:

![image](https://user-images.githubusercontent.com/90405/39302388-20547f50-495b-11e8-99bc-741fcaf6e0e0.png)